sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Trigger.java

```diff
@@ -89,6 +89,9 @@ protected Trigger() {
    * Specifically, triggers that are time based and intended to provide speculative results should
    * continue providing speculative results. Triggers that fire once (or multiple times) should
    * continue firing once (or multiple times).
+   *
+   * <p>This method delegates its behavior to {@link #getContinuationTrigger(List)} which is
```

Comment:
updated.  NB: this design seems kind of odd, since it seems as though if you override getContinuationTrigger() to no longer call getContinuationTrigger(List), you would still need to provide an implementation of getContinuationTrigger(List) which would then never get called?